client/signup/jetpack-connect/index.jsx

```diff
@@ -68,14 +81,29 @@ const JetpackConnectMain = React.createClass( {
 	},
 
 	onURLEnter() {
+		this.props.recordTracksEvent( 'calypso_jpc_url_submit', {
+			jetpack_funnel: this.state.currentUrl,
```

Comment:
Now I'm checking it, this is probably me misinterpreting a comment of @richardmuscat . Richard, can you confirm?